Java/Java.sublime-syntax

```diff
@@ -372,25 +330,14 @@ contexts:
           pop: true
         - match: (?=\S)
           pop: true
-    - match: \b(goto|const)\b
-      scope: invalid.illegal.java
-    - match: \b(try|catch|finally|throw)\b
-      scope: keyword.control.catch-exception.java
     - match: '\?|:'
       scope: keyword.control.java
-    - match: \b(return|break|case|continue|default|do|while|for|switch|if|else)\b
-      scope: keyword.control.java
     - match: \b(instanceof)\b
       scope: keyword.operator.java
-    - match: (==|!=|<=|>=|<>|<|>)
+    - match: (==|!=|<=|>=|<>|(?<!\.)<|>)
```

Comment:
This is a look-behind pattern and incompatible with the internal sregex engine, degrading performance by *a lot*. You should be able to find a way without needing a look-behind.